src/Search_Replace_Command.php

```diff
@@ -47,10 +47,16 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 *
 	 * ## OPTIONS
 	 *
-	 * <old>
+	 * [<old>]
```

Comment:
I don't love that we're changing the behavior of the positional arguments to fix this very small edge case.

Might it be possible to handle this in `wp-cli/wp-cli` somehow? Better handling of positional arguments that look like associative arguments?

Alternatively, is there some other workaround that could be employed for this edge case?

Reply:
It's been quite a long time since I did the tests, but I do remember that I couldn't find a way to make that work properly without using this approach.

Maybe a change under `wp-cli/wp-cli` making it aware of which are the accepted associative args and use it as positional if it does not recognize it?